core-kotlin-modules/core-kotlin-concurrency-3/src/main/java/com/baeldung/delays/Delays.kt

```diff
@@ -0,0 +1,44 @@
+package com.baeldung.delays
```

Comment:
Let's make this package `comparedelays` instead:

```suggestion
package com.baeldung.comparedelays
```